packages/ui-components/tsconfig.hooks.build.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig.build.json", // base that will be ovewrited
```

Comment:
Comments aren’t actually allowed in JSON, so I’m surprised the formatter and linter aren’t flagging this issue.

Reply:
Or, chatgpt leftover :) could you also test this scroll in app?